tools/data_export/export_remote.xml

```diff
@@ -1,10 +1,8 @@
-<tool id="export_remote" name="Export datasets" version="0.1.0" profile="21.05">
+<tool id="export_remote" name="Export datasets" version="0.1.1" profile="21.05">
     <description>to remote files source</description>
-    <!-- TODO
     <requirements>
-        <requirement type="package" version="21.05">galaxy-files</requirement>
+        <requirement type="package" version="24.2">galaxy-files</requirement>
```

Comment:
Thinking about this, this will only work if the container is built with all optional dependencies. That's probably not currently the case ?

Reply:
```
❯ docker run -it quay.io/biocontainers/galaxy-files:24.2.3--pyhdfd78af_0 python -c "import fs.dropboxfs"
WARNING: The requested image's platform (linux/amd64) does not match the detected host platform (linux/arm64/v8) and no specific platform was requested
Traceback (most recent call last):
  File "<string>", line 1, in <module>
ModuleNotFoundError: No module named 'fs.dropboxfs'
```
😢